### PR TITLE
All that was needed is documenting the possibility for JSX

### PR DESCRIPTION
**Pull request by paazmaya**
#31 was much simpler than expected

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +84,11 @@ grunt.initConfig({
 });
 ```
 
+You can also inspect [`jsx` files used in React.js](http://facebook.github.io/react/docs/jsx-in-depth.html),
+simply by using that suffix in the `src` list.
+The command line version of `jsinspect` uses a specific flag for this,
+[namely `--jsx`](https://github.com/danielstjules/jsinspect#usage).
+
```
Comment (stefanjudis) [inline README.md]: I'm not sure I understand this completely.

I have to set files with `.jsx` plus enabling the flag via config?

Comment (paazmaya) [inline README.md]: Only specifying the files is needed.
The command line tool uses a flag for achieving the same behaviour

Comment (stefanjudis) [inline README.md]: > The command line version of `jsinspect` uses a specific flag for this, 
> [namely `--jsx`](https://github.com/danielstjules/jsinspect#usage).

Okay got. :bowtie: 

I'm not sure if that information is valuable for a grunt-user. Usually the people do not care too much about the base modules. :)

Comment (paazmaya) [inline README.md]: I am assuming that once users get comfortable with using Grunt, they might have the curiosity to look deeper...

Unless there is anything else, let's Go!

**Comment by stefanjudis** [comment]
:+1:

**Pull request merged**